packages/core/src/progress-circle/progress-circle.element.spec.ts

```diff
@@ -155,4 +155,160 @@ describe('progress circle element – ', () => {
+      component.value = 80;
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label')).toBe('Loading 80%');
+    });
+    it('should not show default label if a custom label is set', async () => {
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('aria-label')).toBe('Loading');
+      componentUnset.setAttribute('aria-label', 'ohai');
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('aria-label')).toBe('ohai');
+
+      await componentIsStable(component);
+      component.setAttribute('aria-label', 'ohai');
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label')).toBe('ohai');
+      component.value = 88;
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label')).toBe('ohai');
+    });
+  });
+
+  describe('i18n', () => {
+    const newLoading = 'ohai';
+    const newLooping = 'kthxbye';
+
+    it('uses i18n strings by default', async () => {
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label').indexOf(component.i18n.loading) > -1).toBe(
+        true,
+        'has loading i18n string'
+      );
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('aria-label').indexOf(componentUnset.i18n.looping) > -1).toBe(
+        true,
+        'has looping i18n string'
+      );
+    });
+    it('updates looping i18n strings as expected', async () => {
+      await componentIsStable(componentUnset);
+      componentUnset.setAttribute('cds-i18n', `{ "loading": "${newLoading}", "looping": "${newLooping}" }`);
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('aria-label').indexOf(newLooping) > -1).toBe(true);
+    });
+    it('updates loading i18n strings as expected', async () => {
+      await componentIsStable(component);
+      component.setAttribute('cds-i18n', `{ "loading": "${newLoading}", "looping": "${newLooping}" }`);
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label').indexOf(newLoading) > -1).toBe(true);
+    });
+  });
+
+  describe('setAriaAttributes', () => {
+    it('should return expected for indeterminate progress', async () => {
+      await componentIsStable(componentUnset);
+      componentUnset.setAriaAttributes();
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('role')).toBe('img');
+      expect(componentUnset.hasAttribute('aria-valuemin')).toBe(false);
+      expect(componentUnset.hasAttribute('aria-valuemax')).toBe(false);
+      expect(componentUnset.hasAttribute('aria-valuenow')).toBe(false);
+      expect(componentUnset.getAttribute('aria-label')).toBe('Loading');
+
+      componentUnset.ariaLabel = 'howdy';
+      await componentIsStable(componentUnset);
+      expect(componentUnset.getAttribute('role')).toBe('img');
```

Comment:
when the progress does not have a value, it is a "looping"/"spinning"/"indeterminate" progress. per a11y, that is considered an image. it is not `role="progress"` because it has no value, value-min, or value-max.